org.radare.Cutter.yml

```diff
@@ -0,0 +1,51 @@
+id: org.radare.Cutter
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: Cutter
+rename-desktop-file: Cutter.desktop
+rename-icon: cutter
+rename-appdata-file: Cutter.appdata.xml
```

Comment:
You are installing it manually below using $FLATPAK_ID var so rename can be removed.